packages/block-editor/src/layouts/flow.js

```diff
@@ -2,19 +2,85 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import {
+	ToggleControl,
+	__experimentalToolsPanelItem as ToolsPanelItem,
+} from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { getBlockGapCSS, getAlignmentsInfo } from './utils';
 import { getGapCSSValue } from '../hooks/gap';
 import { shouldSkipSerialization } from '../hooks/utils';
+import useSetting from '../components/use-setting';
 
 export default {
 	name: 'default',
 	label: __( 'Flow' ),
-	inspectorControls: function DefaultLayoutInspectorControls() {
-		return null;
+	inspectorControls: function DefaultLayoutInspectorControls( {
+		clientId,
+		defaultControls = {},
+		layout,
+		layoutBlockSupport,
+		onChange,
+	} ) {
+		const defaultThemeLayout = useSetting( 'layout' );
+		const { allowInheriting = true } = layoutBlockSupport;
+
+		// Only show the inherit toggle if it's supported,
+		// a default theme layout is set (e.g. one that provides `contentSize` and/or `wideSize` values),
+		// and either the default / flow or the constrained layout type is in use, as the toggle switches from one to the other.
+		const showInheritToggle = !! (
```

Comment:
Could we keep this logic in the parent layout panel, so as not to duplicate it across two layout types?

Reply:
> Could we keep this logic in the parent layout panel, so as not to duplicate it across two layout types?

Good question! I couldn't work out a way to combine this toggle with the content size controls without moving it down to the layout type level. If we keep them separate (as you suggest in your other comment), then we can happily move it back to the parent.

Let me know if you can think of a better way to go about it, though! (I'll hold off on implementing further changes until we hear more from designers)